spec/support/runner.rb

```diff
@@ -88,6 +88,14 @@ def install_report?
     @options.fetch(:install_report, true)
   end
 
+  def run_env
+    { "APPSIGNAL_PUSH_API_KEY" => @push_api_key }
+  end
+
+  def run_command(_arguments)
+    %(echo "`Runner` subclasses must implement `run_command`"; exit 1)
```

Comment:
You can probably just raise a Ruby error here? It's so broken the test shouldn't even run.

Reply:
Agreed completely 👍 this is a weird workaround for the same outcome. The message even makes more sense as a `NotImplementedError` than as output from the shell.